src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -30,7 +30,59 @@
                 <br>
                 <small class="text-muted">@Model.Review.PackageDisplayName</small>
             </p>
-            <div class="my-1">
+
+            <div class="dropdown d-inline-block my-1">

```

Comment:
I think we can just joing this with the existing pending/ approved indicator since they don't carry out any other function other than say approved or pending. Perhaps they can also be the dropdown for requesting approval. That can be done as a different set of changes though.

Reply:
@chidozieononiwu I'm not fully understanding your suggestion here. If we put the 'Request Review' feature into the 'pending / approved' button, how can we represent both 'pending' state and the ability for people to request review?

Reply:
So will it ever say "pending", or will it instead say "request review"? It seems possible that it can be both of those things at the same time. If it says pending, and there is a drop-down arrow to request reviews - do you think that gives enough affordance to users so that they know the functionality is there?